modules/mesh-layers/src/utils/matrix.js

```diff
@@ -57,8 +58,15 @@ function makeMatrix({
 }
 
 function calculateModelMatrices(attribute) {
-  // eslint-disable-next-line no-invalid-this
-  const {data, getYaw, getPitch, getRoll, getScale, getTranslation, getMatrix} = this.props;
+  const {
+    data,
+    getYaw,
+    getPitch,
+    getRoll,
+    getScale,
+    getTranslation,
+    getTransformMatrix
+  } = this.props;
```

Comment:
Side note: Using `this` here seems wonky to me. It has to be bound to an object elsewhere...